lib/beaker-hostgenerator/hypervisor/hcloud.rb

```diff
@@ -0,0 +1,27 @@
+require 'beaker-hostgenerator/data'
+require 'beaker-hostgenerator/hypervisor'
+require 'deep_merge/rails_compat'
```

Comment:
Why is this needed? I don't see any merging going on.

Reply:
probably a leftover because I started with the docker implementation as base for hcloud. I removed it.